deploy-ocp.yml

```diff
@@ -30,17 +30,17 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   tasks:
-    # Inject "start: false" in the layout to not start any VM yet.
-    # Starting the VM will be done later, either by the tool deploying
-    # OSP, or the one deploy RHOSO.
-    # VM initial configuration, when managed, is done using cloud-init.
-    - name: Ensure no VM is started when we create them during this run
+    # Set amount: 0 for all vms except the ocp ones.
+    # The rest are created by the create-infra playbook, and managing
+    # them from here can cause problems in some environments, e.g
+    # when running each playbook from a different zuul vm
+    - name: Ensure only ocp vms are managed here
       vars:
         _no_start: >-
           {% set _vms = {}                                          -%}
           {% for _type in _cifmw_libvirt_manager_layout.vms.keys()  -%}
           {%   if _type is not match('^ocp.*')                      -%}
-          {%     set _ = _vms.update({_type: {'start': false}})     -%}
+          {%     set _ = _vms.update({_type: {'amount': 0 }})       -%}
```

Comment:
If the idea of the playbook is to reuse a layout that may contains stuff that is not only OCPs isn't safer create an intermediary layout with only the ocps and pass it to the role? No one warranties that the libvirt_manager won't do something with those hosts even if the amount is 0 in the future. Removing them from the layout looks like a less error prone/more maintainable way of passing the layout.

Reply:
fair enough, that was my first version for this PR, let me recover it